packages/valory/contracts/gnosis_safe/contract.py

```diff
@@ -430,6 +431,7 @@ def get_raw_safe_transaction(  # pylint: disable=too-many-arguments,too-many-loc
         configured_gas = base_gas + safe_tx_gas + 75000
         tx_parameters: Dict[str, Union[str, int]] = {
             "from": sender_address,
+            "gas": configured_gas,
```

Comment:
added this to avoid gas estimation when building the transaction, which might lead to GS026 when repricing. This can happen if the node doing the estimation is using the pending state for gas estimation.

Reply:
We cannot supply a block identifier on `buildTransaction()`